client/src/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Instead of importing the whole `ReactDOM` package, why not import what you need.
```import { render } from 'react-dom';```